_posts/2019-10-31-pulp-3-GA-update.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Pulp 3.0 Release Update
+author: Brian Bouterse
+tags:
+  - 3.0
+---
+As announced in [an earlier blogpost](https://pulpproject.org/2019/09/30/pulp-3-GA-timeline/) Pulp
+3.0 is scheduled to be frozen on Nov 12th and Generally Available on Dec 3rd, 2019. Here are details
+on how we'll be building and curating releases for Pulp 3, including 3.0.0 and other releases like
+3.0.1, 3.0.2, ..., 3.1.0, 3.1.1, etc.
+
+### Terminology
+
+* **release**: is a x.y.z number. It's specific because it always includes the "z"
+* **y-release**: is a release where the z portion equals 0, e.g. 3.0.0, 3.1.0, 3.2.0, etc.
+* **z-release**: is a release where the z portion is not 0, e.g. 3.0.1, 3.0.2, 3.1.1, etc.
+* **z-stream**: The set of z releases following a specific y-release.
+
+
+### Release Lead
+
+Each 3.y release will have a release lead who performs the cherry picking of fixes to be included in
+the 3.y.z releases. Brian Bouterse will be the release lead for pulpcore for the 3.0, which includes
+releases 3.0.0, 3.0.1, and any other 3.0 z-stream releases. For 3.1 and its z-stream, another release
+lead will be identified.
+
+Each plugin team is encouraged to have a release lead who performs similar funcitons for that
+plugins y-release. 
```

Comment:
s/funcitons/functions
s/plugins/plugin's

Reply:
ty fixed in next push.